release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -957,11 +974,34 @@ module openconfig-if-ip {
     }
 
     leaf suppress {
+      status deprecated;
       type boolean;
       default false;
       description
-        "When set to true, router advertisement neighbor discovery
-        messages are not transmitted on this interface.";
+        "When set to true, router advertisement neighbor discovery messages
```

Comment:
Changing the description of a leaf that is subsequently deprecated seems an interesting choice. If this leaf is to be removed, then should it not just be that the existing description remains the same and there is a new leaf or set leaves that are created that  implement the modified functionality?

Reply:
Reverted this change